src/gluonts/shell/serve/app.py

```diff
@@ -37,6 +37,7 @@
 
 
 class ForecastConfig(BaseModel):
+    freq: str
```

Comment:
I think this has the downside that now `freq` is required for every requests, even if the given predictor has a fixed `freq`.

I guess one could set it to `None` by default.

Reply:
I am fine with that. I just think that setting this up explicitly will help the user to track quicker what is happening in case something is breaking.

Reply:
Fine with what?